composer.json

```diff
@@ -15,14 +15,14 @@
         }
     },
     "require": {
-        "php": ">=7.0",
+        "php": ">=7.0,<7.4",
```

Comment:
Tests do not work with 7.4, travis also does not include that version – I think it's best to make it work when we'd need that.